home.asm

```diff
@@ -785,7 +785,7 @@ INCLUDE "home/random.asm"
 INCLUDE "home/joypad.asm"
 INCLUDE "home/palettes.asm"
 
-Func_dd4: ; 0xdd4
+HorrendousMultiplyAbyL: ; 0xdd4
```

Comment:
I believe the proper term for this algorithm is peasant multiplication, or [Ancient Egyptian multiplication](https://en.wikipedia.org/wiki/Ancient_Egyptian_multiplication), because of course it is more clever than horrendous.